extensionsGallery-insider.json

```diff
@@ -4827,7 +4827,7 @@
 						}
 					],
 					"statistics": [],
-					"flags": "preview"
+					"flags": ""
```

Comment:
@erinstellato-ms They removed the preview flag from their package.json, so I removed it here too. That work for you, or would you rather we keep it for the ADS version for now?

(note that there's likely to be some small issues with how that's displayed due to different content in gallery vs extension itself if we do keep it. I can go over that separately with you if needed)

Reply:
@Charles-Gagnon Checked VS Code and the extension is NOT in preview there, so it looks like we should remove that.